<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>IonomicsHub_New</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    /* The extra [class] is a hack that increases specificity enough to
       override a similar rule in reveal.js */
    ul.task-list[class]{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      font-size: inherit;
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
  </style>
  <link rel="stylesheet" href="sane_defaults.css" />
</head>
<body>
<h1 id="welcome-to-the-revamped-ionomics-hub">Welcome to the revamped
Ionomics hub</h1>
<p>The IonomicsHub (iHUB) was first accessible to the public in 2007 as
the Purdue Ionomics Information Management System (PiiMS). Hosted at
Purdue University, the system provided integrated workflow control, data
storage, and analysis to facilitate high-throughput ionomic data
collection, along with integrated tools for data search, retrieval,
visualization, and creation of DOIs associated with sets of data <a
href="www.plantphysiol.org/cgi/doi/10.1104/pp.106.092528">(Baxter et
al., 2007)</a>.</p>
<p>Ionomic data collected on Arabidopsis thaliana shoot samples was
actively loaded onto the iHUB from 2002 – 2011. It contained ionomics
data on 205,169 unique A. thaliana samples. This represented
approximately 4,000,000 publically accessible data points from 15,369
unique lines/accessions, and includes 11,478 fast neutron mutagenized
plants, 28,357 T-DNA mutagenized plants (representing 3,452 genes),
21,960 EMS mutagenized plants, and 40,388 wild-type plants including 753
different accessions and 1,474 inbred lines. Over the years the system
was upgraded with the addition of various features (e.g. <a
href="http://dx.doi.org/10.1145/2567948.2577033">Salt et al., 2014</a>),
and extended to host data on 26,268 rice samples and 51,554
Saccharomyces cerevisiae samples, including a genome-wide set of
knockout and over expression strains <a
href="http://www.biomedcentral.com/1471-2164/13/623">(Yu et al.,
2012)</a>. Since it went live the iHUB had 15,423 unique users from 124
countries.</p>
<p>In 2019 the iHUB was moved to the University of Nottingham where it
underwent extensive software upgrades. In 2018 the iHUB domain name
ionomicshub.org was renewed by Purdue University for 5 years.
Unfortunately, when the domain name came up for renewal in 2023 it was
mistakenly not renewed, and was bought by a third party. This third
party was unwilling to sell the domain name back to the iHUB. Owners of
the ionomicshub.org domain are now using the domain to host their own
website, where they are also using the iHUB logo without permission. To
be clear: Despite the use of the iononomicshub logo and scraped text
from the old ionomicshub.org, neither David Salt nor Ivan Baxter have
any association with the current ionomicshub.org. The current owners do
not have the data they are claiming.</p>
<p>Unfortunately, the ionomicshub.org address was hardcoded into the
underlying code, so the full ionomicshub system crashed and we have been
unable to resurrect it. To continue to provide community access to the
ionomics data that was hosted at the iHUB, we have deposited the
underlying datasets at Zenodo and linked them from this site using the
ionomicshub.work domain. Here, we provide links for download of files
containing ionomic data from the iHUB on <em>A. thaliana</em> and <em>S.
cerevisiae</em>. This site no longer provides the ability to upload
data, or perform searches, visualization or data set and DOI generation.
The ionomicshub.work domain is owned by Ivan Baxter and it is our
intention to maintain this site in perpetuity as a reference for the
community. If we are able to recover additional data from the original
ionomicshub.org or if others have deposited additional ionomics datasets
on public repositories, we will be happy to add links to that data.</p>
<p>File of Arabidopsis thaliana shoot ionomic data</p>
<pre><code>1. Elemental concentrations (showing the isotope used for quantification) normalized to sample weight, units ppm. Data shown as median values . Metadata associated with samples including growth conditions, line name and genetic background also shown.

2. Zscore (showing the isotope used for quantification) normalized to either average of wild-type genetic background or sample set average, using weight normalised data. Data shown as median values. Metadata associated with samples including growth conditions, line name and genetic background also shown.

3. Elemental concentrations (showing the isotope used for quantification) normalized to sample weight, units ppm for leaf tissue of A. thaliana 1001 collection with replicate samples, and normalization, as described in Campos et al., (2021)(DOI @ : https://onlinelibrary.wiley.com/doi/10.1111/tpj.15177). 

4. Elemental concentrations (showing the isotope used for quantification) normalized to sample weight, units ppm for seed tissue of A. thaliana 1001 collection with replicate samples, and normalization, as described in Campos et al., (2021).

5. Elemental concentrations (showing the isotope used for quantification) for leaf and seed tissue of A. thaliana 1001 showing a single fully normalized value for each accession as described in Campos et al., (2021). Data downloaded from https://ffionexplorer.nottingham.ac.uk/ionmap/ 29/06/2025.</code></pre>
<p>Files of Saccharomyces cerevisiae ionomic data</p>
<pre><code>1. Elemental concentrations (showing the isotope used for quantification) showing individual replicate samples in units of ppb divided by the optical density of the culture and normalized using the method described in Yu et al., (2012) for genome-wide knockout (haploid and diploid) and overexpression collection

2. Elemental concentrations (showing the isotope used for quantification) showing zstat score for each line (replicates not shown) normalized using the method described in Yu et al., (2012) for genome-wide knockout (haploid and diploid) and overexpression collection.</code></pre>
<h1 id="yeast-files"><strong>Yeast files:</strong></h1>
<p><a
href="https://ddpsc-my.sharepoint.com/:x:/g/personal/rdhakal_danforthcenter_org/Efj-pTmVEQxItxFUhoT7R5wBuhuIulvv5LS-Tvjkb1ZNCQ?download=1">Yeast
Nstat table</a></p>
<pre><code>- Normal stat scores for Yeast</code></pre>
<p><a
href="https://ddpsc-my.sharepoint.com/:x:/g/personal/rdhakal_danforthcenter_org/ERWjHxV94LtBlJqxHygRsFoBMXV3PHJnY5VSsLoSPmOsLA?download=1">Yeast
Ostat table</a></p>
<pre><code>- Observed stat scores for Yeast </code></pre>
<p><a
href="https://ddpsc-my.sharepoint.com/:x:/g/personal/rdhakal_danforthcenter_org/EQfd2WbfWMxPuxIUMhhxD5UBcjP0vfegr5pnKh8lvv9wlA?download=1">Yeast
Zstat table</a></p>
<pre><code>- Zscore stat scores for Yeast   </code></pre>
<p><strong>Arabidopsis</strong></p>
<p><a
href="https://ddpsc-my.sharepoint.com/:x:/g/personal/rdhakal_danforthcenter_org/EZxN77t2VRBAv9RtdHyicGAB2sCAh-vZbNgqb3-M5s-COQ?download=1">Link
to Zscore table</a></p>
<p><strong>File Store</strong></p>
<table>
<colgroup>
<col style="width: 0%" />
<col style="width: 75%" />
<col style="width: 6%" />
<col style="width: 1%" />
<col style="width: 15%" />
</colgroup>
<thead>
<tr>
<th>File_ID</th>
<th>Description</th>
<th>File Location</th>
<th>Category</th>
<th>Link to file</th>
</tr>
</thead>
<tbody>
<tr>
<td>43</td>
<td>Ionomics data and QTL results of 10 grow-outs of the Maize IBM
population. Published in Asaro et al. 2016.</td>
<td>Asaroetal2016.MaizeIBMionomicsandQTLdata.zip</td>
<td>Maize</td>
<td>https://www.dropbox.com/scl/fi/d5dnlgccnd7zvvb8hxzu5/Asaroetal2016.MaizeIBMionomicsandQTLdata.zip?rlkey=vv4vaewj1jtn2cg35zz3fnmcj&amp;dl=0</td>
</tr>
<tr>
<td>16</td>
<td>R scripts and data from Ziegler et al. Plant Genome, 2012</td>
<td>Ziegler_etal_PlantGenome2012_ScriptsAndData.zip</td>
<td>Soybean</td>
<td></td>
</tr>
<tr>
<td>41</td>
<td>R scripts from Shakoor et al. Sorghum GWAS 2016</td>
<td>Shakooretal2016.Scripts.zip</td>
<td>Other</td>
<td></td>
</tr>
<tr>
<td>46</td>
<td>Ionomics data, Joint Linkage and GWAS results from 2006 North
Carolina, Florida, New York and Puerto Rico grow-outs of Maize NAM
population.</td>
<td>MaizeNAMionomicsandGWASdata.zip</td>
<td>Maize</td>
<td></td>
</tr>
<tr>
<td>12</td>
<td>All ionomics data from the Clayton, NC 2006 NAM grow out.
Fields:plate= 48 well plate seeds are loaded into. Unique in
experimentnuminset= unique ID within each WR setRun= sample # on ICP
unique within each WR setFullnum= sample number within the whole
experimentICP.run= ICP run - usually 576 samples per runWR.run= WR run -
288 samplesWR.plate= plate within the WR run, 6 48-well
plates/WR.runICP.plate= plate within the ICP run 12 48-well
plates/ICP.runICP.runnum= sample number on ICP unique within each ICP
runMissing numbers in ICP.runnum,Run, and fullnum are usually the
locations Controls were run at.All data is weight-normalized ppm
concentrations.</td>
<td>MaizeNAM.ClaytonNCdataset.csv</td>
<td>Maize</td>
<td></td>
</tr>
<tr>
<td>15</td>
<td>Ionomics data from the Florida 2006 NAM grow out.Fields:plate= 48
well plate seeds are loaded into. Unique in experimentnuminset= unique
ID within each WR setRun= sample # on ICP unique within each WR
setFullnum= sample number within the whole experimentICP.run= ICP run -
usually 576 samples per runWR.run= WR run - 288 samplesWR.plate= plate
within the WR run, 6 48-well plates/WR.runICP.plate= plate within the
ICP run 12 48-well plates/ICP.runICP.runnum= sample number on ICP unique
within each ICP runMissing numbers in ICP.runnum,Run, and fullnum are
usually the locations Controls were run at.All data is weight-normalized
ppm concentrations.</td>
<td>MaizeNAM.06FLdataset.csv</td>
<td>Maize</td>
<td></td>
</tr>
<tr>
<td>14</td>
<td>Purdue 2010 IBM, Association Panel, and Ky21 Maize population
ionomics data.Fields:plate= 48 well plate seeds are loaded into. Unique
in experimentnuminset= unique ID within each WR setRun= sample # on ICP
unique within each WR setFullnum= sample number within the whole
experimentICP.run= ICP run - usually 576 samples per runWR.run= WR run -
288 samplesWR.plate= plate within the WR run, 6 48-well
plates/WR.runICP.plate= plate within the ICP run 12 48-well
plates/ICP.runICP.runnum= sample number on ICP unique within each ICP
runMissing numbers in ICP.runnum,Run, and fullnum are usually the
locations Controls or samples from another population were runAll data
is weight-normalized ppm concentrations.</td>
<td>IBM.AssPan.Ky21.10PUdataset.csv</td>
<td>Maize</td>
<td></td>
</tr>
<tr>
<td>13</td>
<td>Purdue 2009 IBM and Association Panel Maize ionomics
data.Fields:plate= 48 well plate seeds are loaded into. Unique in
experimentnuminset= unique ID within each WR setRun= sample # on ICP
unique within each WR setFullnum= sample number within the whole
experimentICP.run= ICP run - usually 576 samples per runWR.run= WR run -
288 samplesWR.plate= plate within the WR run, 6 48-well
plates/WR.runICP.plate= plate within the ICP run 12 48-well
plates/ICP.runICP.runnum= sample number on ICP unique within each ICP
runMissing numbers in ICP.runnum,Run, and fullnum are usually the
locations Controls were run at.All data is weight-normalized ppm
concentrations.</td>
<td>IBM.AssPan.09PUdataset.csv</td>
<td>Maize</td>
<td></td>
</tr>
<tr>
<td>17</td>
<td>All public ionomics data from the Arabidopsis PiiMs in CSV
format</td>
<td>Arabidopsis_PublicDataReport.zip</td>
<td>Arabidopsis</td>
<td>this is the 5 tables we found</td>
</tr>
<tr>
<td>42</td>
<td>Raw data, scripts, tables and figures for "Canopy position has a
profound effect on soybean seed composition"</td>
<td>FinalDataAndScripts.zip</td>
<td>Soybean</td>
<td>not needed</td>
</tr>
<tr>
<td>23</td>
<td>R scripts and data from Baxter et al. PNAS 2008</td>
<td>Baxter_etal_supp_scriptsandinputfiles.zip</td>
<td>Arabidopsis</td>
<td>https://www.dropbox.com/scl/fi/syc3ojhgdrlaflkqfreg2/Baxter_etal_supp_scriptsandinputfiles.zip?rlkey=wud8ho8ceql1s6w1zmaj3st96&amp;st=p8pjc6yu&amp;dl=0</td>
</tr>
<tr>
<td>11</td>
<td>R scripts and data from Baxter et al. PLoS ONE, 2012</td>
<td>Baxter_etal_PLoSONE_2012_data_scripts.zip</td>
<td>Arabidopsis</td>
<td>https://www.dropbox.com/scl/fi/ohe59mnw0oda6zem3r758/Baxter_etal_PLoSONE_2012_data_scripts.zip?rlkey=cpweftt6bggbhthy7dmmx5yed&amp;st=2e3uuqns&amp;dl=0</td>
</tr>
<tr>
<td>51</td>
<td>Scripts and data for 'Multivariate analysis reveals environmental
and genetic determinants of element covariation in the maize grain
ionome. Asaro et
al. https://www.biorxiv.org/content/early/2017/12/31/241380</td>
<td>Multivariate_Analysis_of_Maize_Grain_Ionome_Data_and_Scripts.zip</td>
<td>Maize</td>
<td>https://www.dropbox.com/scl/fi/nkurujk6qygnshjh0dam4/Multivariate_Analysis_of_Maize_Grain_Ionome_Data_and_Scripts.zip?rlkey=t5uetw810u3k86t41mtaq66wi&amp;dl=0</td>
</tr>
<tr>
<td>53</td>
<td>IPSRI.2015.2016.IonomicsData.csv contains measurements of elemental
concentrations for the Illinois Protein Strain Recombinant Inbred
(IPSRI) population grown at UIUC in 2015 and 2016. There is a row for
each seed measured, multiple seeds were measured per plot.
Concentrations were normalized for sample weight and corrected for batch
effects by extracting the residuals from this linear model:Phenotype ~
batch + kernelWeight. Thus, each phenotype can be interpreted as
concentration (in parts-per-billion) above or below the population mean.
Extreme analytical outliers have been removed from this dataset. The
SingleSeedWeight is the weight of the kernel digested and analyzed for
ionomics in milligrams. This data was generated as part of the NSF award
IOS-1638507</td>
<td>IPSRI.2015.2016.IonomicsData.zip</td>
<td>Maize</td>
<td>https://www.dropbox.com/scl/fi/kbh46wmftqvjxmrv4mw3f/2015.2016.IPSRI.Ionomics.Data.zip?rlkey=3r7gkip4yl6rf7cl6wko90zea&amp;dl=0</td>
</tr>
<tr>
<td>50</td>
<td>Final Data and Scripts from Huber et al, PeerJ 2016 "Canopy position
has a profound effect on soybean seed composition"</td>
<td>FinalDataAndScripts081816.zip</td>
<td>Soybean</td>
<td>https://www.dropbox.com/scl/fi/gltosbkp71lovstcp15tn/FinalDataAndScripts081816.zip?rlkey=uelxytv59s0h5jv22uxwy8gq0&amp;st=hyy88593&amp;dl=0</td>
</tr>
<tr>
<td>24</td>
<td>Scripts and Data files for Baxter et al. 2014.
http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0087628</td>
<td>ScriptsandData.zip</td>
<td>Maize</td>
<td>https://www.dropbox.com/scl/fi/9qlqmocxtx3fzzyqmjux3/ScriptsandData.zip?rlkey=l8idkgiueilhitrzlnbshi1tw&amp;st=r6fh3pc7&amp;dl=0</td>
</tr>
<tr>
<td>52</td>
<td>Scripts and Data files for Kohler et al. "Increased temperatures may
safeguard the nutritional quality of crops under future elevated CO2
concentrations"</td>
<td>FInalKohler_Scriptsanddatafiles.zip</td>
<td>Soybean</td>
<td>https://www.dropbox.com/scl/fi/4yzeovpkqemaghp50sfiq/FInalKohler_Scriptsanddatafiles.zip?rlkey=zztsbcq34i8otcxa3t4xi3igh&amp;dl=0</td>
</tr>
</tbody>
</table>
</body>
</html>
